### ISSUE TITLE
Is there any production application out there?

### ISSUE THREAD

**Issue by workless** [qmlweb/qmlweb#422]
I was wondering if it has been used in production since i can only see relative simple demos.

**Comment by stephenmdangelo** [comment]
Hello, I have created a production application that uses a shared QML code base for a C++ deployment and a web deployment. So it is possible :)

The biggest issues I’ve run into are performance related (it just takes a lot of time to go through and evaluate all of the property bindings on a large tree of items) and an eventual crash in Safari when creating/destroying a lot of items within Loaders. 

My application has a series of “tabs” with varying content on each. I originally tried to load them all at once, but that was far too slow. I then tried a Loader that would load each tab when it was selected, but that would cause an eventual crash in Safari, I’m guessing due to leaking memory somewhere but debugging led me nowhere. 

So I replaced the Loader with an IFrame and that made things work well enough.
